utils/scripts/compute-impacted-libraries.py

```diff
@@ -63,18 +63,22 @@ def main() -> None:
             if user_choice is None:
                 # user let the script pick impacted libraries
                 if match:
-                    result.add(match[2])
+                    if not file.endswith((".md", ".rdoc", ".txt")):
```

Comment:
Good logic, but not at the good location : it must be at the top of the for loop, and skip everything if it's a `.md` ...